errbot/backends/irc.py

```diff
@@ -416,6 +430,21 @@ def on_dcc_connect(self, dcc, event):
     def on_dcc_disconnect(self, dcc, event):
         self.transfers.pop(dcc)
 
+    def on_endofnames(self, c, e):
```

Comment:
Also, `c` and `e` are pretty bad names for parameters, although I'm guessing you copied them from the IRC library directly.

What would they stand for? If I had to guess, `callback` and `event`? Using such names might be more clear.